build/mixins-documentation.js

```diff
@@ -6,9 +6,11 @@ const Stream = require('stream')
 sh.config.fatal = true
 
 const stylelintLineRegex = /\s*?\/\/\s?stylelint.*/
+const fileUrl = 'https://github.com/twbs/bootstrap/blob/master/'
```

Comment:
Not master :P

Reply:
I used master as placeholder before asking this: https://github.com/twbs/bootstrap/pull/26500#issuecomment-432535399

But I underestimated your speed of reviewing 😀